helios/models.py

```diff
@@ -1564,7 +1562,8 @@ def process(self, linked=True, check_dupes=True, preferred_encoding=None):
         if self.voter_file_content:
             voter_data = base64.decodestring(self.voter_file_content.encode())
         else:
-            voter_data = open(self.voter_file.path, "rb").read()
+            with open(self.voter_file.path, "rb") as voter_data:
+                voter_data.read()
```

Comment:
I think this line should look like line `1529` - saving result of read to the `voter_data`.